src/main/DynamicConfig.ts

```diff
@@ -589,6 +589,23 @@ export class DynamicConfig implements IDynamicConfig {
     }
 
     private async loadConfigs(): Promise<IRootConfigValue> {
+        const envConfigFile: ILoadedFile =
+            await this.configLoader.loadEnvironment()
+
+        console.log({ envConfigFile })
+
+        const envConfig: IRootConfigValue = ConfigBuilder.createConfigObject(
+            {
+                type: 'local',
+                name: envConfigFile.name,
+            },
+            this.translator(envConfigFile.config),
+        )
+
+        if (Array.isArray(envConfigFile.extends)) {
```

Comment:
If this config does extend other files we short circuit loading the default config.